### PR TITLE
updated all that node endpoint url

### PR DESCRIPTION
**Pull request by jiseongnoh**
Allthatnode changed their RPC endpoint URL to https://klaytn-mainnet.g.allthatnode.com/full/evm. 
Source: https://www.allthatnode.com/klaytn.dsrv

## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ x] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/kaiachain/kaia-docs/blob/main/CONTRIBUTING.md)
- [ ] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribute
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/references/service-providers/public-en.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Since we cannot guarantee uptime and stability of the endpoints, do not use them
 |---|---|---|---|
 |[Kaia Foundation](https://www.klaytn.foundation)|`https://public-en-cypress.klaytn.net`|klay,eth,net|Full|
 ||`https://archive-en.cypress.klaytn.net`|klay,eth,net|Archive|
-|[All That Node](www.allthatnode.com)|`https://klaytn-mainnet-rpc.allthatnode.com:8551`|klay,eth,net|Full|
+|[All That Node](www.allthatnode.com)|`https://klaytn-mainnet.g.allthatnode.com/full/evm`|klay,eth,net|Full|
```
Comment (ayo-klaytn) [inline docs/references/service-providers/public-en.md:28]: Thanks @jiseongnoh for the update.
Can you also change testnet to: `https://klaytn-baobab.g.allthatnode.com/full/evm`?

**Comment by ayo-klaytn** [comment]
In the meantime @jiseongnoh can you comment the statement below?

I have read the CLA Document and I hereby sign the CLA

**Comment by jiseongnoh** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/references/service-providers/public-en.md (hunk 1 above)
Comment (jiseongnoh) [reply docs/references/service-providers/public-en.md:28]: Thank you. I updated it.

**Review by ayo-klaytn — APPROVED** [review]

**Review by scott-klaytn — APPROVED** [review]

**Pull request merged**